openassessment/xblock/test/test_allow_resubmission.py

```diff
@@ -0,0 +1,159 @@
+"""This module contains the tests for the allow_resubmission module."""
+
+import datetime
+import unittest
+
+import ddt
+
+from openassessment.xblock.utils.allow_resubmission import allow_resubmission
+
+
+class ConfigDataMock:
+    """Mock class for the ORAConfigAPI object."""
+
+    def __init__(self):
+        self.allow_learner_resubmissions = True
+        self.submission_due = "2029-01-01T00:00:00+00:00"
+        self.resubmissions_grace_period_hours = 0
+        self.resubmissions_grace_period_minutes = 0
```

Comment:
Same question here about the values. Is this just a default?

Reply:
Yes, it is the default value. In other tests, it is tested with other values